robottelo/constants.py

```diff
@@ -797,8 +797,6 @@
     'Edit hosts',
     'Edit partition tables',
     'Red Hat Access Logs',
-    'Remote Execution Manager',
-    'Remote Execution User',
```

Comment:
For GA, Remote execution plugin will be installed by default.  If these roles are not present by default, this is perhaps a bug.